hazelcast/src/main/java/com/hazelcast/internal/config/ConfigValidator.java

```diff
@@ -240,6 +240,15 @@ private static void warnForDeprecatedHotRestartProp(MapConfig mapConfig,
         }
     }
 
+    public static void warnForUsageOfDeprecatedSymmetricEncryption(Config config, ILogger logger) {
+        if (config.getNetworkConfig() == null || config.getNetworkConfig().getSymmetricEncryptionConfig() == null) {
+            return;
+        }
+        if (config.getNetworkConfig().getSymmetricEncryptionConfig().isEnabled()) {
+            logger.warning("Symmetric encryption is deprecated and may be vulnerable to security problems. Consider TLS instead");
```

Comment:
a missing dot at the end of the 2nd sentence? 
also the word is rather scary. consider toning it down.

Reply:
tuned down